core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -1133,7 +1138,8 @@ void addOrderItem(List<SqlNode> orderByList, RelFieldCollation field) {
                   RelFieldCollation.NullDirection.UNSPECIFIED);
         }
       }
-      orderByList.add(toSql(field));
+      SqlNode orderByNode = toSql(field);
```

Comment:
is this expanded just for readability

Reply:
Think it's a lingering artifact of previous debugging lol, i'll get that changed up